### DISCUSSION TITLE
Show Actions context menu with mouse based hotkey

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1788] [Suggestions]
DC offers a command that allows user to be able to show only the “Actions” context menu.
cm_ContextMenu
justcm_openmenu=no

I was thinking, being that user can also access context menu using RMB, that maybe a modifier can be added to also allow access to this “Actions” context menu as well.
Not sure if conflicts with anything but maybe something “Ctrl+RMB”
Might not be useful but just an idea.

Thank You

**Comment by milnak** [comment]
This would be fantastic! It currently requires a bunch of browsing a menu to find that "Actions" popout.

![image](https://github.com/user-attachments/assets/91487474-5d68-46a5-85df-fa720456de86)
 
I'd even be fine with (an option for) Right click to default to the action menu if it applies to a file, and then at the bottom of a menu an option for "Context Menu >" which would bring up the Windows context menu.

**Comment by r5to** [comment]
Maybe this can help you.
https://github.com/doublecmd/doublecmd/discussions/2103#discussioncomment-12265763

**Comment by milnak** [comment]
I assigned that to Ctrl+F10 and it works great. Thanks!
